### DISCUSSION TITLE
How is it different from obsidian comparison?

### DISCUSSION THREAD

**Discussion by niaiwomattt** [TriliumNext/Trilium#7628] [Q&A] [answered]
### Describe feature

And it is not yet mature, and the document introduction does not highlight the characteristics of this software.

**Comment by salmund** [comment, marked as answer]
Trilium is a completely different application. Having used Obsidian for a year, I have to say that I find it much easier to organise myself with Trilium.

What I like is this database system, everything is in one file, you can easily use SQL and everything is easily found.

On Obsidian, the advantage is to have folders and files locally, but it is not necessarily very practical for research (even if there is DataView).

After all, it all depends on your needs, and the advantage of these two applications is that they can be extended as you wish.

(BTW, this thread should be in "discussions" and not in "issues" I think)

**Comment by lamka02sk** [comment]
I like Trilium because it is more user friendly and easier to work with than Obsidian. Also Trilium is much more powerful and easier to create custom scripts. 
There is just one drawback that made me switch to Obsidian recently - no mobile support. Yes, there is mobile version, but my experience with it is pretty bad. It is buggy and looks pretty bad.
